src/net/test.rs

```diff
@@ -138,52 +256,117 @@ impl TestServerBuilder {
     }
 }
 
-pub struct TestClients(pub [MpcHelperClient; 3]);
-
-impl Deref for TestClients {
-    type Target = [MpcHelperClient; 3];
-
-    fn deref(&self) -> &Self::Target {
-        &self.0
-    }
+fn get_test_certificate_and_key(id: HelperIdentity) -> (&'static [u8], &'static [u8]) {
+    // TODO(640): to be removed when we standardize on rustls
+    #[cfg(not(target_os = "macos"))]
+    let key = TEST_KEYS[id];
+    #[cfg(target_os = "macos")]
+    let key = TEST_KEYS_MUNGED[id];
+    (TEST_CERTS[id].as_bytes(), key.as_bytes())
 }
 
-impl From<TestClients> for [MpcHelperClient; 3] {
-    fn from(value: TestClients) -> [MpcHelperClient; 3] {
-        value.0
-    }
+#[must_use]
+pub fn get_test_identity(id: HelperIdentity) -> ClientIdentity {
+    let (certificate, private_key) = get_test_certificate_and_key(id);
+    ClientIdentity::Certificate(Identity::from_pkcs8(certificate, private_key).unwrap())
 }
 
-impl TestClients {
-    /// Build default set of test clients
-    ///
-    /// All three clients will be configured with the same default server URL, thus,
-    /// at most one client will do anything useful.
-    pub fn default() -> Self {
-        Self::builder().build()
-    }
-
-    /// Return a test client builder
-    pub fn builder() -> TestClientsBuilder {
-        TestClientsBuilder::default()
-    }
-}
+pub const TEST_CERTS: [&str; 3] = [
```

Comment:
Is this cleaner as `const X: [&[u8]; 3] = [b"...", b"...", b"..."];`?

Reply:
I guess so... I tend to think of PEM as string-like and prefer to put it in string types, but it does seem like the stuff we're passing these to prefers bytes.